src/main/java/com/lothrazar/cyclicmagic/item/equipbauble/ItemCharmAir.java

```diff
@@ -96,7 +97,13 @@ public void onTick(ItemStack stack, EntityPlayer player) {
   }
 
   @Override
-  public IRecipe addRecipe() {
-    return super.addRecipe(new ItemStack(Blocks.BONE_BLOCK));
+  public boolean getIsRepairable(ItemStack par1ItemStack, ItemStack par2ItemStack)
+  {
+    return par2ItemStack.getItem() == craftItem.getItem();
```

Comment:
This doesnt consider metadata or nbt so for example it woill match coal and charcoal.  . which is ok for the simple items we're using, but ill probably just change it after merge